automaxlair/MaxLairInstance.py

```diff
@@ -438,30 +447,39 @@ def check_shiny(self) -> bool:
                                          (180, 255, 255), 10
                                          )
 
-    def check_stats(self,
-                    check_attack: bool,
-                    attack_stat: int,
-                    check_speed: bool,
-                    speed_stat: int
-                    ) -> bool:
+    def check_stats(self) -> bool:
         """Detect whether a Pokemon has perfect stats.
         """
-        ret = True
-        if check_attack:
-            if str(attack_stat) not in self.read_text(self.get_frame(), self.attack_stat_rect, threshold=False, segmentation_mode='--psm 8'):
-                ret = False
-                self.log('attack is not good', 'DEBUG')
-            else:
-                self.log('attack is good', 'DEBUG')
 
-        if check_speed:
-            if str(speed_stat) not in self.read_text(self.get_frame(), self.speed_stat_rect, threshold=False, segmentation_mode='--psm 8'):
-                ret = False
-                self.log('speed is not good', 'DEBUG')
+        # First check if the attack stat match one of the expected value
+        isAttackMatching = True
```

Comment:
I think it's convention to use lower case and underscores in variables (as opposed to camel case).